frontend/src/components/user-management/student/AddClassroomModal.tsx

```diff
@@ -34,13 +41,22 @@ const AddClassroomModal = (): React.ReactElement => {
     formState: { errors },
   } = useFormContext<ClassroomForm>();
   const { onOpen, onClose, isOpen } = useDisclosure();
+  const [className, setClassName] = useState("");
+  const [schoolYear, setSchoolYear] = useState(0);
+  const [gradeLevel, setGradeLevel] = useState(Grade.K);
+  const [teacher, setTeacher] = useState("6347056d47cd96025c18e639");
+  const [testSessions, setTestSessions] = useState([]);
```

Comment:
```suggestion
```
 i missed this as well from last PR, but can we remove testSessions from `ClassRequestDTO`. it is always empty during creation and we can't update it in the edit classroom modal. You will need to update BE and initialize field to [] in `createClass`

 We will need to add a new endpoint for `AddTestSessionToClass(classId, testSessionId)` - i added a new ticket in the backlog for now

Reply:
@joyce-shi pls correct me if i'm missing smth